android/app/src/main/java/app/organicmaps/downloader/DownloaderAdapter.java

```diff
@@ -386,9 +386,7 @@ private void processClick(boolean clickOnStatus)
         }
         case CountryItem.STATUS_FAILED ->
         {
-          RetryFailedDownloadConfirmationListener listener =
-              new RetryFailedDownloadConfirmationListener(mActivity.getApplication());
-          MapManager.warn3gAndRetry(mActivity, mItem.id, listener);
```

Comment:
hmm what was its role and why its no longer needed?

Reply:
RetryFailedDownloadConfirmationListener was needed to remove "Download failed" notification. It's no longer needed as the new progress notification will replace it.